src/components/Tracker/elements/Encounters/Encounters.tsx

```diff
@@ -96,11 +99,31 @@ const Encounters = React.memo(function Encounters() {
   const handleScroll = (index: number) => {
     listRef.current?.scrollToItem(index + 1, 'center');
   };
+  
+  const navigate = useNavigate();
+
+  const handleOpen = (e: React.MouseEvent<HTMLDivElement, MouseEvent>, index: number) => {
+    e.preventDefault();
+    e.stopPropagation();
+    navigate(`/badgedetail/${selectedGame?.value}/${index}`);
+  };
+
+  const handleKeyPress = (e: React.KeyboardEvent<HTMLDivElement>, index: number) => {
+    if (e.key === 'Enter' || e.key === ' ') {
```

Comment:
There are multiple different char codes for space across keyboard layouts. It might be better to look for another method of checking said keypress

Reply:
Changed to use the key code instead of the actual key